core-httpclient-impl/src/main/java/com/optimizely/ab/OptimizelyFactory.java

```diff
@@ -369,11 +419,44 @@ public static Optimizely newDefaultInstance(ProjectConfigManager configManager,
             .withApiManager(odpApiManager != null ? odpApiManager : new DefaultODPApiManager())
             .build();
 
+        DefaultCmabClient defaultCmabClient = new DefaultCmabClient(CmabClientConfig.withDefaultRetry());
```

Comment:
Can we add one more interface - 
```suggestion
    public static Optimizely newDefaultInstance(ProjectConfigManager configManager, NotificationCenter notificationCenter, EventHandler eventHandler, ODPApiManager odpApiManager, CmabService cmabService) {
```